packages/hooks/use-popup-manager/index.ts

```diff
@@ -0,0 +1,30 @@
+import { ref, getCurrentInstance } from 'vue'
+import { useGlobalConfig } from '../use-global-config'
+
+let zIndex = 0
+export const usePopupManager = () => {
+  let globalInitialZIndex = 2000 // TODO: move to @element-plus/constants
+
+  const setGlobalInitialZIndex = (value: number) =>
+    (globalInitialZIndex = value)
+
+  const getInitialZIndex = () => {
+    if (!getCurrentInstance()) return ref(globalInitialZIndex)
+    return useGlobalConfig('zIndex', globalInitialZIndex)
+  }
```

Comment:
**Question:** is it possible that we use `computed` here instead of invoking the function ourselves?

E.g.

```ts
const globalConfig = useGlobalConfig('zIndex', globalInitialZIndex)

const initialZIndex = computed(() => {
  if (!getCurrentInstance()) return globalInitialZIndex
  return globalConfig.value
})
```